repos/system_upgrade/el7toel8/actors/logshandler/actor.py

```diff
@@ -0,0 +1,19 @@
+from leapp.actors import Actor
+from leapp.libraries.actor.library import prepare_log_environment
+from leapp.tags import IPUWorkflowTag, FactsPhaseTag
+
+
+class LogsHandler(Actor):
+    """
+    Prepare logs directories structure, archive and remove old logs from previous Leapp run.
+
+    This actor prepare the system to new logs and archive the old ones for debugging pourposes.
+    """
+
+    name = 'logs_handler'
+    consumes = ()
+    produces = ()
+    tags = (IPUWorkflowTag, FactsPhaseTag.Before)
```

Comment:
@pirat89, any thoughts?

Reply:
I don’t know, but shouldn’t majority of this not be handled by rpm? Directory creation especially? Also this could be done by the main upgrade entry point

Reply:
Yes, directory creation should be in rpm spec. What do you mean by main upgrade entry point?

Reply:
I think @vinzenz suggest to add code in  https://github.com/oamg/leapp/blob/master/leapp/cli/upgrade/__init__.py#L101 instead create a new actor.

Reply:
New PR in Leapp framework opened to solve this.
https://github.com/oamg/leapp/pull/493